python/lsst/obs/decam/ingest.py

```diff
@@ -160,6 +159,7 @@ def getInfo(self, filename, filetype="raw"):
                 info[self.instcalPrefix] = self.expnumMapper[expnum][self.instcalPrefix]
                 info[self.dqmaskPrefix] = self.expnumMapper[expnum][self.dqmaskPrefix]
                 info[self.wtmapPrefix] = self.expnumMapper[expnum][self.wtmapPrefix]
+
         elif filetype == "raw":
             md = afwImage.readMetadata(filename, self.config.hdu)
             phuInfo = self.getInfoFromMetadata(md)
```

Comment:
Maybe remove the two lines above, and move the following six lines to after line 172 (the call to the base method)?